tsconfig.build.json

```diff
@@ -18,6 +18,6 @@
     "useUnknownInCatchVariables": false,
     "outDir": "./build" 
   },
-  "include": ["src/**/*", "src/routes, examples/chatbot"],
+  "include": ["src/**/*", "src/routes", "examples/chatbot", "test/**/*"],
```

Comment:
IMHO, including `examples/chatbot` in the `tsconfig.json` may cause confusion, as it requires running `yarn install` in that directory as well. This can be unclear, especially if I only intend to run the SA.